mgmtsystem_nonconformity_hr/models/mgmtsystem_nonconformity_hr.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2019 Marcelo Frare (Ass. PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+# Copyright 2019 Stefano Consolaro (Ass. PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class MgmtsystemMgmHR(models.Model):
+    """
+    Extend nonconformity adding fields for department
+    """
+
+    _inherit = ['mgmtsystem.nonconformity']
+
+    # new fields
+    # department reference
```

Comment:
```suggestion
```
self explainatory ;) no need for comments

Reply:
like before, ok